### ISSUE TITLE
ExtensionFromMime returns jpe for image/jpeg

### ISSUE THREAD

**Issue by rodineijf** [dart-lang/mime#55]
Even that's not wrong the `.jpe` extension is less used than `.jpg` or `.jpeg`.

Should we change the order of `jpe` in the defaultExtensionMap?

**Comment by Holidaybear** [comment]
Is there any update?

**Comment by mx1up** [comment]
yes, last week I applied the latest review suggestions and am awaiting approval now
